README.MD

```diff
@@ -47,6 +47,7 @@ programmer among salesmen etc?
   - Will you be promoted in technical fields or maybe management/business ones? How does company see this?
   - Will you be able to change between different development fields, like changing languages, technologies etc?
 - What are the factors for promotion? Results or rather time spent in the company?
+- Where do you see the company in the next 5/10 years?
```

Comment:
There is already 
`What are the company's plans for next 1-2 years and how it will affect your work?`

You can edit this point
`What are the company's plans for next 1-2-5-10 years and how it will affect your work?`

Reply:
ok